backbone-nested-models.js

```diff
@@ -150,7 +150,9 @@
 
       _.each(this.relations, function(rel, key) {
         if (_.has(attrs, key)) {
-          attrs[key] = attrs[key].toJSON();
+          if(attrs[key] !== null) {
```

Comment:
@alexanderott thanks for the PR! I don't actively use this package anymore, so I don't know if this would be correct as you have done it, or would it be better to move the conditional up to the previous statement:

```js
if (_.has(attrs, key) && attrs[key] !== null) {
```